utilities_common/helper.py

```diff
@@ -35,6 +37,15 @@ def get_port_acl_binding(db_wrap, port, ns):
     return acl_tables
 
 
+def validate_interface_name_length(iface_name):
+    """
+    Verify that interface name length does not exceed IFNAMSIZ
+    """ 
+    if len(iface_name) == 0:
+        return False
+    return True if len(iface_name) >= IFNAMSIZ else False
```

Comment:
@arfeigin the function should return True in case the validation was successful (`ifName` does not exceed the `IFNAMSIZ`). Is this a typo?

Reply:
Typo indeed, thanks a lot!